src/js/components/Map.js

```diff
@@ -174,6 +184,7 @@ export default class Map extends Component {
       response.map.graphics.clear();
       //- Attach events I need for the info window
       response.map.infoWindow.on('show, hide, set-features, selection-change', mapActions.infoWindowUpdated);
```

Comment:
@KaylaKremer Maybe listen for on(`hide`.. with a different callback that simply Clears Out the selectedFeatureTitles?